Formula/abyss.rb

```diff
@@ -0,0 +1,30 @@
+class Abyss < Formula
+  desc "Genome sequence assembler for short reads"
+  homepage "http://www.bcgsc.ca/platform/bioinfo/software/abyss"
+  url "https://github.com/bcgsc/abyss/releases/download/2.0.2/abyss-2.0.2.tar.gz"
+  sha256 "d87b76edeac3a6fb48f24a1d63f243d8278a324c9a5eb29027b640f7089422df"
+
+  depends_on "boost" => :build
+  depends_on "google-sparsehash" => :build
+  depends_on :mpi => :cc
+
+  resource("testdata") do
+    url "http://www.bcgsc.ca/platform/bioinfo/software/abyss/releases/1.3.4/test-data.tar.gz"
+    sha256 "28f8592203daf2d7c3b90887f9344ea54fda39451464a306ef0226224e5f4f0e"
+  end
+
+  def install
+    system "./configure", "--enable-maxk=128", "--prefix=#{prefix}",
+           "--disable-dependency-tracking", "--disable-silent-rules",
+           "--disable-openmp"
+
+    system "make"
```

Comment:
Is the separate `make` step actually needed?

Reply:
No.